src/main/java/Main.java

```diff
@@ -0,0 +1,92 @@
+import java.util.Scanner;
+
+
+public class Main {
+    public static final int MAX_TASK_CAPACITY = 100;
+    public static final String EXIT = "bye";
+    public static final String PRINT_TASK_LIST = "list";
+    public static final String TASK_DONE = "done";
+    public static final String ADD_EVENT = "event";
+    public static final String ADD_DEADLINE = "deadline";
+    public static final String ADD_TODO = "todo";
+
+
+    public static void main(String[] args) {
+
+        printGreetingMessage();
+
+        Scanner in = new Scanner(System.in);
+        Task[] tasks = new Task[MAX_TASK_CAPACITY];
+        String command;
+        int numTask = 0;
+
+        while (true) {
+            command = in.nextLine();
+
+            if (command.equals(EXIT)) {
+                break;
+            } else if (command.equals(PRINT_TASK_LIST)) {
+                Task.printTaskList(tasks);
+                continue;
+            }
+            //the above are single word command
+
+            numTask = respondMultiWordCommand(tasks, command, numTask);
+        }
+
+
+        printExitMessage();
+    }
+
+    private static int respondMultiWordCommand(Task[] tasks, String command, int numTask) {
+        String[] words = command.split(" ");
+        if (words[0].equals(TASK_DONE)) {
+            tasks[Integer.parseInt(words[1]) - 1].markAsDone();
+        } else {
+            numTask = addTask(words[0], tasks, command, numTask);
+        }
+        return numTask;
+    }
+
+    private static void printExitMessage() {
+        System.out.println("____________________________________________________________");
```

Comment:
Maybe you should consider making this a static constant? It would help tidy up the repetition in the code.